net-dialup/freeradius/freeradius-3.0.19.ebuild

```diff
@@ -0,0 +1,242 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
```

Comment:
This is not an issue now, but we are looking to deprecate python2.7 from Gentoo. You should really start pushing upstream / looking for patches to enable python3 support for this.

Reply:
Hi, as I wrote in IRC channel this is the last release with py2.7 in Gentoo. 3.0.20 will be available with vanilla + patch (for enabling py3 only).

Reply:
Good, good. As I said it's not a stopper now, but you ought'd to have a plan for future.

Reply:
Just as a reference to the discussion: 
https://github.com/FreeRADIUS/freeradius-server/pull/3185
and here the POC for the next ebuild:
https://github.com/geaaru/geaaru_overlay/blob/master/net-dialup/freeradius/freeradius-3.0.20.ebuild#L67